{
  "id": "CVE-2021-46179",
  "sourceIdentifier": "cve@mitre.org",
  "published": "2023-08-22T19:16:21.883",
  "lastModified": "2023-08-22T20:10:36.537",
  "vulnStatus": "Awaiting Analysis",
  "descriptions": [
    {
      "lang": "en",
      "value": "Reachable Assertion vulnerability in upx before 4.0.0 allows attackers to cause a denial of service via crafted file passed to the the readx function."
    }
  ],
  "metrics": {},
  "references": [
    {
      "url": "https://github.com/upx/upx/issues/545",
      "source": "cve@mitre.org"
    }
  ]
}